doc/Interns/vassenx/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+May 6th, 2020
```

Comment:
This is great.  I forgot to mention.  Can you add the time to this file too please?